_config_authoring.yml

```diff
@@ -129,29 +147,48 @@ defaults:
     values:
       ucp_org: "docker"
       ucp_repo: "ucp"
-      ucp_version: "3.0.0-beta2"
+      ucp_version: "3.1.1"
+  - scope: # This is a bit of a hack for the get-support.md topic.
+      path: "ee"
+    values:
+      ucp_org: "docker"
+      ucp_repo: "ucp"
+      dtr_repo: "dtr"
+      ucp_version: "3.1.1"
+      dtr_version: "2.6.0"
+      dtr_latest_image: "docker/dtr:2.6.0"
```

Comment:
@thaJeztah should be `2.6.1` for latest version, yeah? FWIW, I don't think anyone's been using `_config_authoring.yml` for the last release, but it will be nice to keep this up-to-date and usable for the next one. It was my understanding that this was an alternate config file for incremental changes and previewing those changes on your local build. Or did I miss the point entirely?

Reply:
Ah; yes, it's updated now that https://github.com/docker/docker.github.io/pull/8010 is merged.

> I don't think anyone's been using _config_authoring.yml for the last release

Yeah, it's likely not often used; we should either remove it (or make sure to keep it in sync)

I'll do a quick rebase and sync it again

Reply:
updated :+1: